sa/model.go

```diff
@@ -589,7 +590,7 @@ func SelectAuthzsMatchingIssuance(
 		issued.Add(1*time.Second),  // leeway for clock skew
 		identifierTypeToUint[string(identifier.TypeDNS)],
```

Comment:
This needs to be changed to use the actual identifier type from the input argument. In fact, it needs more change than that, since the sql query currently believes that all the identifiers have the same type.